try-it.md

```diff
@@ -535,17 +535,22 @@ Sometimes you may want to look directly at Ironic to debug something.
 The metal3-dev-env repository contains a clouds.yaml file with
 connection settings for Ironic.
 
-metal3-dev-env will install the openstack command line tool on the
-provisioning host as part of setting up the cluster. The openstack tool
-will look for clouds.yaml in the current directory or you can copy it to
-~/.config/openstack/clouds.yaml. Version 3.19.0 or higher is needed to
-interact with Ironic using clouds.yaml.
+Metal3-dev-env will install the unified OpenStack and standalone 
+OpenStack Ironic command-line clients on the provisioning host as 
+part of setting up the cluster. Please make sure to export
+```CONTAINER_RUNTIME``` environment variable before you execute
+commands.
```

Comment:
Are the 2 sentences in this paragraph related somehow?

Reply:
Seems not really. Updated it, thanks.